### PR TITLE
Added APU sound, MCDU button sound, and extractor fan self test sound

### PR DESCRIPTION
**Pull request by hotshotp**
## Summary of Changes

Completely overhauled APU sound -
Fully redone interior startup run and shutdown sequence (only heard in cabin)
Fully redone exterior startup run and shutdown sequence with igniters and fuel flow sounds

Redone the initial startup of the engine before fuel introduction just to make it less painful to listen to

Added new MCDU button sounds

and finally added avionics fan self test to the exterior (blower extractor fan) - only self test for now, the run sound will be included later (its still realistic that it stops after the self test as the valve does close in cold temps and therefore makes no sound, so just pretend like there is cold temps right now)

## References

All sounds are recorded from real a320s by komp. This specific APU is an APS3200. Real life startup speeds might not match as the current apu startup and shutdown is way too slow.

https://discord.com/channels/738864299392630914/738864300818432023/793596859116814376

## Additional context

Discord username (if different from GitHub):

Boris#9134

## Testing instructions

Start ext power and listen to exterior
Start apu and listen from the cabin and exterior view
Listen in cabin view when starting engine
Click mcdu buttons

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,8 @@
 1. [FLIGHTMODEL] Preliminary fix for flaps 1 unrecoverable dives - @donstim (donbikes#4084)
 1. [UTILS] Added Vfe and Vfen table - @MisterChocker (Leon)
 1. [PFD] Fixed wrong Vfe and Vfen indication - @MisterChocker (Leon)
+1. [SOUND] Reworked interior and exterior apu sound - @hotshotp (Boris#9134)
+1. [SOUND] Added new MCDU button sounds
```
Comment (pepperoni505) [inline .github/CHANGELOG.md:20]: Add credit to this line

**Review by pepperoni505 — APPROVED** [review]
LGTM

**Comment by Highflyer2010** [comment]
Category: QA Tester
Name: Discord User Highflyer2010#8932
Date of testing: Current date 30/12/2020
Version of the sim: 1.12.13.0
PR Tested: #2801

Object of test:APU sound, MCDU button sound, and extractor fan self test sound

Testing technique: Loaded Cold and dark to listen for the new sounds of the APU, MCDU buttons ad extractor fan

Steps you've made to test the Pull Request:

1. Loaded Sim and loaded Aircraft C&D
 
2. Spawned at EGNT at a gate

3. Turned on External Power
 
4. Changed View to Exterior for external power sound

5. Went Onto the MCDU and Pressed Buttons

6. Turned APU on Listened from Cabin & Exterior

7. Started Engines


Flight notes (If required):N/A

Issues (If Any): 
MCDU Buttons - Didn't hear any changes from the sound already within the aircraft

Overall Rating: Good

Conclusions of the testing:

Testing of the EXT PWR Sound - No Problems
APU Sound Exterior - No Problems
Cabin View Engine Start - No Problems

Love the new sounds i never heard any difference on the MCDU

**Comment by sidnov** [comment]
Category: QA Tester
Name: sidnov#8337
Date of testing: 30/12/2020
Version of the sim: 1.12.13.0
PR Tested: #2801
Object of test: APU sound, MCDU button sound, and extractor fan self test sound

Testing technique: Spawn C/D at gates, normal startup

Steps you've made to test the Pull Request:
1. Spawned at VABB Gate 5 C/D
2. Turned on battery, external power. Listened to sounds in external view, passenger window view and cockpit view
3. Turned on APU, switched views to listen to the sounds. APU sound in external view is much more prominent and realistic.
4. Turned on engines, observed sounds in external, passenger views. 
5. Turned off engines, APU and external power, made the cockpit silent
6. Tested MCDU button sounds



Issues (If Any): None
Overall Rating: Good
Conclusions of the testing: The APU sounds are the highlight of this PR. So much more energy into it. That abrupt sound/note change on engine startup is fixed too. Good one Boris! :)

**Comment by tjburni** [comment]
Category: QA Tester
Name: [Z+1]tjburni#0409
Date of testing: Current date 30.12.2020
Version of the sim:1,12,13
PR Tested: #2801
Object of test:APU sound, MCDU button sound, and extractor fan self test sound

Testing technique: tested sounds
Steps you've made to test the Pull Request: full C&D IFR
1. started at gate
2. Started ext power and listened to exterior sounds
3. Started apu and listened from the cabin and exterior view
4. Listened in cabin view when starting engine
5. Clicked mcdu buttons 

Flight notes (If required):

Issues (If Any):none 
Overall Rating: Good
Conclusions of the testing:The new sounds are really great.Thanks for the amazing work

**Comment by hotshotp** [comment]
> Love the new sounds i never heard any difference on the MCDU

fixed :)

**Review by kberg93 — APPROVED** [review]
Category: QA Tester
Name: Kberg#0666
Date of testing: Current date 12/30/2020
Version of the sim: 1.12.13.0
PR Tested: #2801 
Object of test: Added APU sound, MCDU button sound, and extractor fan self test sound

Testing technique: Clicking and starting up/shut down apu.

Steps you've made to test the Pull Request: 
1. APU Startup internal and external sound good.
2. MCDU buttons sounds good.
3. Blower test on exterior sound good.

Flight notes (If required): Parked at gate only

Issues (If Any): Nope.

Overall Rating: Good.

Conclusions of the testing: Great Success :boratemoji:

Screenshots:

**Comment by Highflyer2010** [comment]
I get nervice when i pick a fault on a PR Glad you got it sorted buddy great PR though them sounds are out of this world such a difference 💯

**Comment by hotshotp** [comment]
> I get nervice when i pick a fault on a PR Glad you got it sorted buddy great PR though them sounds are out of this world such a difference 💯 

Thank you!! Appreciate it!

**Comment by BritishAvgeek** [comment]
Merged so I won't worry about doing the full report but it all worked very well - the APU noise was fab. I tested it 5hrs ago now and Boris was watching the stream and noted an issue with the MCDU buttons which he fixed of course - the APU noises are just oooooft

**Pull request merged**